### PR TITLE
chore: Update CODEOWNERS

### PR DESCRIPTION
**Pull request by Geoff-Ford**
👋 

Hey guys! Forgot to remove myself here.

Can somebody approve and merge please.

Cya

### CODE REVIEW

**Review by kamiranoff — APPROVED** [review]
👋

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # The following people are responsible for this project and its live instances:
-*    @aronshamash @Geoff-Ford @kamiranoff
+*    @aronshamash @kamiranoff
```
Comment (aronshamash) [inline CODEOWNERS:2]: not many code-owners....

**Review by aronshamash — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (kamiranoff) [reply CODEOWNERS:2]: yes, I am pretty sure some people could be added.

Comment (aronshamash) [reply CODEOWNERS:2]: bye @Geoff-Ford

Comment (Geoff-Ford) [reply CODEOWNERS:2]: 👋  
I can't merge as don't have write access anymore.

**Pull request merged**